spec/factories/memberships.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: memberships
+#
+#  id              :integer          not null, primary key
+#  organization_id :integer
+#  user_id         :integer
+#  send_email      :boolean
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  org_admin       :boolean
+#
+
+FactoryGirl.define do
+  factory :membership do
+    organization
+    user
+    send_email false
+    org_admin false
+
+    factory :membership_org_admin do
```

Comment:
I'm not sure this will work. If anything, you'll want to move this factory outside to be at the same depth as the membership factory.

Again, I'd probably use a trait for this

``` 
trait :admin do
  org_admin true
end
```

I don't think you'd need the association, because it's already associated back to a User